cmake/FindLuaJIT.cmake

```diff
@@ -0,0 +1,63 @@
+# Locate LuaJIT library
+# This module defines
+#  LUAJIT_FOUND, if false, do not try to link to Lua
+#  LUA_LIBRARIES
+#  LUA_INCLUDE_DIR, where to find lua.h
+#  LUAJIT_VERSION_STRING, the version of Lua found (since CMake 2.8.8)
+
+## Copied from default CMake FindLua51.cmake
```

Comment:
Any chance that can be merged with FindLua.cmake? It looks partially duplicated.

Reply:
I'm not fully convinced that this is really a good idea.

Both lua and luajit are independent packages with their own lifecycle. Once a new lua version arrives, FindLua.cmake can be simply replaced by a new stock version (I think Lua projects provides their own FindLua.cmake, but I don't know if this is used here).

Although they partially look similar, they reference different include files and libraries in different places. I fell merging FindLuaJIT into FindLua might complicate the latter too much.

Reply:
That's a fair point. In this case the FindLuaJIT script should be at least consistent about the variables it exports. Either make them all LUAJIT_* or LUA_*. The latter would keep the rest of the cmake file simpler.

Other than that, the PR looks alright to me. I haven't tested on other OSses, though.

Reply:
Thank you for your feedback. I have changed all variable names to LUAJIT_* now, as I was a bit worried to introduce naming clashes otherwise. That might be an issue in particular for variable LUA(_JIT)_FOUND, which derived from the package name "Lua" or "LuaJIT" respectively.